test/functional/p2p_permissions.py

```diff
@@ -22,12 +22,19 @@ def set_test_params(self):
         self.extra_args = [[],[]]
 
     def run_test(self):
+
         self.checkpermission(
-        # relay permission added
-        ["-whitelist=127.0.0.1", "-whitelistrelay"],
+        # default permissions (no specific permissions)
+        ["-whitelist=127.0.0.1"],
         ["relay", "noban", "mempool"],
         True)
 
+        self.checkpermission(
+        # relay permission removed (no specific permissions)
+        ["-whitelist=127.0.0.1", "-whitelistrelay=0"],
+        ["noban", "mempool"],
```

Comment:
Generally we intend with 4 spaces after an opening `(` that is immediately followed by a new line

Reply:
you mean that you want me to move the comment above?